addons/hr/models/hr_employee_base.py

```diff
@@ -31,7 +31,7 @@ class HrEmployeeBase(models.AbstractModel):
     mobile_phone = fields.Char('Work Mobile', compute="_compute_work_contact_details", store=True, inverse='_inverse_work_contact_details')
     work_email = fields.Char('Work Email', compute="_compute_work_contact_details", store=True, inverse='_inverse_work_contact_details')
     work_contact_id = fields.Many2one('res.partner', 'Work Contact', copy=False)
-    related_contact_ids = fields.Many2many('res.partner', 'Related Contacts', compute='_compute_related_contacts')
+    related_contact_ids = fields.Many2many('res.partner', compute='_compute_related_contacts')
```

Comment:
I believe that if the string was added, it was probably for that specific purpose of having the "s", since there might be multiple contacts. For me, the best fix would be to name each argument
```suggestion
    related_contact_ids = fields.Many2many(comodel_name='res.partner', string="Related Contacts",
        compute='_compute_related_contacts')
```

Reply:
My pleasure :D

Indeed, I was not so sure about the string or not, but it makes more sense to keep it then, since it won't have a big impact on the readability of the code, but will make more sense when the label is displayed in a view

I think the first parameter is very explicit and in the standard code, only around 1% of the `Many2many(` are followed by comodel_name=, so I think the best thing to do for both world would be 

`fields.Many2many('res.partner', string="Related Contacts",` ...

What do you think @Bertrand2 ? :smile:

Reply:
Seems ok to me :)